src/geo/gmaps/cartodb-layer-group-base.js

```diff
@@ -90,20 +94,21 @@ CartoDBLayerGroupBase.prototype.clear = function () {
 CartoDBLayerGroupBase.prototype.update = function (done) {
   var self = this;
   this.loading && this.loading();
-  this.getTiles(function(urls, err) {
-    if(urls) {
-      self.tilejson = urls;
-      self.options.tiles = urls.tiles;
-      self.tiles = 0;
-      self.cache = {};
-      self._reloadInteraction();
-      self.refreshView();
-      self.ok && self.ok();
-      done && done();
-    } else {
-      self.error && self.error(err)
-    }
-  });
+
+  var tilejson = self.model.get('urls');
+  if(tilejson) {
+    self.tilejson = tilejson;
+    self.options.tiles = tilejson.tiles;
+    self.tiles = 0;
+    self.cache = {};
+    self._reloadInteraction();
+    self.refreshView();
+    self.ok && self.ok();
+    done && done();
+  } else {
+    self.error && self.error('URLs have not been fetched yet');
```

Comment:
the self is not really necessary anymore, since the code is not located in a closure, perhaps we can tidy up this?